micropipenv.py

```diff
@@ -351,6 +355,16 @@ def _get_requirement_info(requirement):  # type: (ParsedRequirement) -> Dict[str
     # Editable requirements.
     editable = getattr(requirement, "editable", False) or getattr(requirement, "is_editable", False)
 
+    # Check for unsupported VCS.
```

Comment:
I am thinking about how to make this check better without duplicates. What about something like:
```python
link = getattr(requirement, "requirement", None) or getattr(requirement, "link", None)
if link and link.startswith(("hg+", "svn+", "bzr+")):
    raise NotSupportedError(…)
```

Reply:
Sounds reasonable, I've adjusted the PR in 33ecbf6 :+1: